redirect/routes.php

```diff
@@ -149,7 +149,11 @@ class Route {
         ],
         'crypto'           => [
             'description' => 'Kryptographie',
-            'target'      => 'https://www7.in.tum.de/um/courses/crypto/ws2021/',
+            'target'      => 'https://www.sec.in.tum.de/i20/teaching/ss-2023/kryptografie',
+        ],
+        'crypto-moodle'    => [
+            'description' => 'Kryptographie Moodle',
+            'target'      => 'https://www.moodle.tum.de/course/view.php?id=86223',
```

Comment:
This adds a redirect to the moodle page using https://mcrypto.tum.sexy

```suggestion
            'target'      => 'https://www.sec.in.tum.de/i20/teaching/ss-2023/kryptografie',
            'moodle_id' => '86223',
        ],
```